Makefile

```diff
@@ -412,13 +418,19 @@ clean-mount-cache: ## Prune all docker mount cache (used by sccache)
 clean-artifacts: ## Remove all dumped artifacts from test runs (see dump_artifacts.py)
 	rm -Rf test_artifacts
 
+.PHONY: run-registry
+run-registry: 
+	nomad/local/local_grapl_registry.sh
+
 .PHONY: push
 push: ## Push Grapl containers to Docker Hub
```

Comment:
Worth updating this comment; we're not pushing anything to Docker Hub.

Reply:
Would this not push to Docker Hub by default?

Reply:
If another registry isn't part of the container names, yes. However, we aren't doing anything with Docker Hub, so it's misleading.

Reply:
I'm not sure what change I should make here then. If this isn't going to push to docker hub, it sounds like it should either:

* be removed
* updated to say "pushed to supplied DOCKER_REGISTRY"

Which makes the most sense?

Reply:
I think Option 2 sounds good for now... whether or not Option 1 should be eventually adopted depends on some stuff that @inickles-grapl and I are tossing around.